### DISCUSSION TITLE
Generated frame ?

### DISCUSSION THREAD

**Discussion by Nebuluss** [volotat/SD-CN-Animation#45] [General]
Hello, could i know where are the generated frames going ?

I have crashed after few hours of a very long video and would like to avoid to run all again

**Comment by volotat** [comment]
For now only videos are saved in order to not clutter up the disk memory. I will probably add global settings in the next update with an ability to save resulting frames as images into a folder alongside with the output video.

**Comment by volotat** [comment]
Check out the last update. There is an option to save the frames into a folder now.
